modules/cudaobjdetect/include/opencv2/cudaobjdetect.hpp

```diff
@@ -162,6 +162,22 @@ class CV_EXPORTS_W HOG : public Algorithm
                         std::vector<Point>& found_locations,
                         std::vector<double>* confidences = NULL) = 0;
 
+    CV_WRAP inline virtual void detect(InputArray img,
```

Comment:
> inline virtual

Please remove "virtual" from these "inline" calls.

Reply:
Removed, sorry I missed those.

Reply:
Am I right in assuming that removing `inline` will mean these don't appear in the shared lib?

Reply:
Exporting in shared libraries is mainly controlled by CV_EXPORTS`.
In this case - through `class CV_EXPORTS_W HOG` declaration.

`inline` is fine here (`virtual` is not).